src/main/java/oss/fosslight/api/controller/v1/ApiOssController.java

```diff
@@ -110,23 +112,47 @@ public CommonResult getOssInfoByDownloadLocation(
     public CommonResult getLicenseInfo(
     		@RequestHeader String _token,
     		@ApiParam(value = "License Name", required = true) @RequestParam(required = true) String licenseName){
-		
+
 		// 사용자 인증
 		userService.checkApiUserAuth(_token);
 		Map<String, Object> resultMap = new HashMap<String, Object>();
-		
+
 		try {
 			List<Map<String, Object>> content = apiOssService.getLicenseInfo(licenseName);
-		
-			
+
+
 			if (content.size() > 0) {
 				resultMap.put("content", content);
 			}
-			
+
 			return responseService.getSingleResult(resultMap);
 		} catch (Exception e) {
 			return responseService.getFailResult(CoConstDef.CD_OPEN_API_UNKNOWN_ERROR_MESSAGE
 					, CoCodeManager.getCodeString(CoConstDef.CD_OPEN_API_MESSAGE, CoConstDef.CD_OPEN_API_UNKNOWN_ERROR_MESSAGE));
 		}
     }
+
+	@ApiOperation(value = "Register New Oss", notes = "신규 Oss 등록")
+	@ApiImplicitParams({
+			@ApiImplicitParam(name = "_token", value = "token", required = true, dataType = "String", paramType = "header")
+	})
+	@PostMapping(value = {API.FOSSLIGHT_API_OSS_REGISTER})
+	public CommonResult registerOss(
+			@RequestHeader String _token,
+			@ApiParam(value = "Oss Mater", required = true) @RequestBody(required = true) OssMaster ossMaster){
+
+		if(userService.isAdmin(_token)) {
+			Map<String, Object> resultMap = new HashMap<String, Object>();
+			try {
+				resultMap = ossService.saveOss(ossMaster);
+				resultMap = ossService.sendMailForSaveOss(resultMap);
+				return responseService.getSingleResult(resultMap);
+			} catch (Exception e) {
+				return responseService.getFailResult(CoConstDef.CD_OPEN_API_UNKNOWN_ERROR_MESSAGE
+						, CoCodeManager.getCodeString(CoConstDef.CD_OPEN_API_MESSAGE, CoConstDef.CD_OPEN_API_UNKNOWN_ERROR_MESSAGE));
+			}
+		}
+		return responseService.getFailResult(CoConstDef.CD_OPEN_API_PERMISSION_ERROR_MESSAGE
+				, CoCodeManager.getCodeString(CoConstDef.CD_OPEN_API_MESSAGE, CoConstDef.CD_OPEN_API_PERMISSION_ERROR_MESSAGE));
```

Comment:
```suggestion
		return responseService.getFailResult(CoConstDef.CD_OPEN_API_PERMISSION_ERROR_MESSAGE,
				CoCodeManager.getCodeString(CoConstDef.CD_OPEN_API_MESSAGE, CoConstDef.CD_OPEN_API_PERMISSION_ERROR_MESSAGE));
```

Reply:
What about conveying the message that this code is not an admin?
We'll have to talk about this later.

Reply:
I tried to create a separate 403 message.
However, there was already a Permission Error so I used this.
Can you tell me exactly what this CD_OPEN_API_PERMISSION_ERROR_MESSAGE means?

Reply:
I didn't realize there were no 403 errors.
CD_OPEN_API_PERMISSION_ERROR_MESSAGE would be a good choice.